tests/resources/test_user.py

```diff
@@ -16,3 +44,115 @@ def test_user_collection_creation():
     session = FakeSession()
 
     assert session == UserCollection(session).session
```

Comment:
We'll need to figure out what the best UX is for instantiating a UserCollection. It could be a peer to projects `citrine.users`. Since this isn't a core feature though, I don't mind keeping it like this for right now.

Reply:
So rn it _is_ a peer to `citrine.projects`.
That aligns nicely with the structure of the API.

Reply:
Oh gotcha. I think I was misled by this line that instantiates it separately.